editor/js/Menubar.Add.js

```diff
@@ -54,12 +74,12 @@ Menubar.Add = function ( editor ) {
 	var option = new UI.Row();
 	option.setClass( 'option' );
 	option.setTextContent( 'Plane' );
-	option.onClick( function () {
+	option.onClick( function() {
 
-		var geometry = new THREE.PlaneGeometry( 2, 2 );
+		var geometry = new THREE[ buffered ? "PlaneBufferGeometry" : "PlaneGeometry" ]( 2, 2 );
```

Comment:
At this point, I think I would just change them to use BufferGeometries directly (without the checkbox).